playwright/tests/challenges.spec.ts

```diff
@@ -213,8 +216,20 @@ test.describe("Hacker challenge speedrun", () => {
     });
     await expect(challengeNinePage.passwordImage).toBeVisible();
 
+    // take screenshot from website
+    await page.screenshot({ path: 'revealed_image_with_password.png' });
+
+    // use Tesseract worker to recognize the text that is displayed on the website
+    // https://github.com/tesseract-ocr/tesseract
+    const { data: { text } } = await (await worker).recognize('revealed_image_with_password.png');
+    console.log('### Recognized text on website:\n', text);
+
+    // extract the password from the text
+    const extractedPassword = getStatedPasswordIn(text)
+    console.log('### Extracted password using OCR:\n', extractedPassword);
+    
     // fill in password and proceed
-    await challengeNinePage.passwordInput.fill(challengeNinePage.password);
```

Comment:
We can delete `challengeNinePage.password` from pages file

Reply:
Deleted